packages/fast-usdc-deploy/src/start-fast-usdc.core.js

```diff
@@ -131,6 +134,7 @@ export const startFastUSDC = async (
   const privateArgs = await deeplyFulfilledObject(
     harden({
       agoricNames,
+      customFeeConfigs,
```

Comment:
hm. we won't be using this code again in mainnet.

I added [a note in #294](https://github.com/Agoric/agoric-private/issues/294#issuecomment-2777198335) to follow up.